src/stateless/dialog_button.vue

```diff
@@ -20,7 +20,7 @@
 import DialogButton from './dialog_button.vue'
 
 export default {
-    name: 'DialogButton', // eslint-disable-line vue/name-property-casing
+    name: 'dialog-button',
```

Comment:
It would be nice to be consistent when it comes to meta names. Why is this and support-text kebab case while others are camel case?